### PR TITLE
Generate openssl methodsynopses based on stubs

### PR DESCRIPTION
**Pull request by kocsismate**

### CODE REVIEW

File: reference/openssl/functions/openssl-cms-encrypt.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- $Revision$ -->
+<refentry xml:id="function.openssl-cms-encrypt" xmlns="http://docbook.org/ns/docbook">
+ <refnamediv>
+  <refname>openssl_cms_verify</refname>
+  <refpurpose>Verify signature</refpurpose>
```
Comment (kocsismate) [inline reference/openssl/functions/openssl-cms-encrypt.xml:6]: the refpurpose of the new functions is not good yet... I've just realized that the refname here is also wrong.

**Comment by kocsismate** [comment]
It's getting better! I've just added more change log entries, and removed more `resource` references. I and removed the stub documentation for `openssl_cms_*` functions which I added, as I've just noticed that had already been committed to master.

**Comment by cmb69** [comment]
Thank you!

File: reference/openssl/functions/openssl-csr-get-public-key.xml [hunk 2]
```diff
@@ -41,9 +41,32 @@
...
+     <row>
+      <entry>&Version;</entry>
+      <entry>&Description;</entry>
+     </row>
+    </thead>
+    <tbody>
+     <row>
+      <entry>8.0.0</entry>
+      <entry>
+       On success, this function returns a <classname>OpenSSLAsymmetricKey</classname> instance now;
```
Comment (kocsismate) [inline reference/openssl/functions/openssl-csr-get-public-key.xml:62]: I've just noticed that I use the wrong article before `OpenSSLAsymmetricKey`. Thanks for fixing it (among other issues I suppose)

**Pull request closed without merging**